c/src/CMakeLists.txt

```diff
@@ -66,8 +69,9 @@ if (NOT (UNIX AND APPLE)) # CPPJouls does not support macos right now
 endif()
 
 # LibGit
-set(USE_REGEX "builtin")
+option(USE_SSH OFF)
```

Comment:
Nice 👍 
I wonder if the "unpushed changes" feature works still, without SSH. Could you maybe add a test for that or shortly test it locally?

Reply:
`USE_SSH` is [off by default](https://github.com/libgit2/libgit2/blob/21a351b0ed207d0871cb23e09c027d1ee42eae98/CMakeLists.txt#L30). I only made it more explicit here in case libgit ever wants to change the defaults.